### PR TITLE
feat: added delay option for the review buttons

### PR DESCRIPTION
**Pull request by 4Source**
Added the option to enable a delay for the review buttons like I mentioned in #996 would be great if someone could provide feedback and/or could test it.

### CODE REVIEW

File: src/lang/locale/ar.ts [hunk 1]
```diff
@@ -80,6 +80,9 @@ Note that this setting is common to both Flashcards and Notes. : أدخل مسا
     FLASHCARD_EASY_DESC: '"تخصيص التسمية للزر "سهل',
     FLASHCARD_GOOD_DESC: '"تخصيص التسمية للزر "جيد',
     FLASHCARD_HARD_DESC: '"تخصيص التسمية للزر "صعب',
+    REVIEW_BUTTON_DELAY: "Button press delay",
+    REVIEW_BUTTON_DELAY_DESC:
+        "Add a delay to the review buttons('Easy', 'Good', 'Hard', 'Show answer') before there could be pressed again.",
```
Comment (st3v3nmw) [inline src/lang/locale/ar.ts:85]: before _they_ could be pressed again

File: src/lang/locale/cz.ts [hunk 2]
```diff
@@ -80,6 +80,9 @@ Note that this setting is common to both Flashcards and Notes.`,
     FLASHCARD_EASY_DESC: 'Customize the label for the "Easy" Button',
     FLASHCARD_GOOD_DESC: 'Customize the label for the "Good" Button',
     FLASHCARD_HARD_DESC: 'Customize the label for the "Hard" Button',
+    REVIEW_BUTTON_DELAY: "Button press delay",
+    REVIEW_BUTTON_DELAY_DESC:
+        "Add a delay to the review buttons('Easy', 'Good', 'Hard', 'Show answer') before there could be pressed again.",
```
Comment (st3v3nmw) [inline src/lang/locale/cz.ts:85]: Same issue here and in the other translation files

File: src/lang/locale/en.ts [hunk 3]
```diff
@@ -80,6 +80,9 @@ Note that this setting is common to both Flashcards and Notes.`,
     FLASHCARD_EASY_DESC: 'Customize the label for the "Easy" Button',
     FLASHCARD_GOOD_DESC: 'Customize the label for the "Good" Button',
     FLASHCARD_HARD_DESC: 'Customize the label for the "Hard" Button',
+    REVIEW_BUTTON_DELAY: "Button press delay",
```
Comment (st3v3nmw) [inline src/lang/locale/en.ts:83]: We could also indicate the units as well:

> Button press delay (ms)

File: src/settings.ts [hunk 4]
```diff
@@ -929,15 +956,15 @@ export class SRSettingTab extends PluginSettingTab {
...
-        containerEl.createEl("hr").insertAdjacentHTML("beforeend");
-
-        // Copyright notice
-        const copyright_paragraph = containerEl.createEl("p");
-        copyright_paragraph.addClass("sr-small-font");
-        copyright_paragraph.insertAdjacentHTML("beforeend", `
-            <em>Shell commands</em> plugin Copyright &copy; 2021 - 2023 Jarkko Linnanvirta. This program comes with ABSOLUTELY NO WARRANTY. This is free software, and you are welcome to redistribute it under certain conditions. See more information in the license: <a href="${GitHub.license}">GNU GPL-3.0</a>.
-        `);     */
+		// Documentation link & GitHub links
+		containerEl.createEl("hr").insertAdjacentHTML("beforeend");
```
Comment (st3v3nmw) [inline src/settings.ts:960]: Curious why the indentation has changed

**Review by st3v3nmw — CHANGES_REQUESTED** [review]

File: src/settings.ts (hunk 4 above)
Comment (4Source) [reply src/settings.ts:960]: I just have run the format not sure could be undone if you like?

**Review by st3v3nmw — APPROVED** [review]
LGTM, thanks.

Please have a look at the merge conflict.. I can't resolve it on the UI.

**Review by st3v3nmw — COMMENTED** [review]
Wait, I think the rebase / merge messed something up. I can't see the code that actually does the delay.. It was there before.

**Comment by 4Source** [comment]
Agree this must be lost during the renaming

**Comment by st3v3nmw** [comment]
> Agree this must be lost during the renaming

Thanks.

**Pull request merged**